### PR TITLE
Organised the code by fixing indentation and  created separate folders

### PR DESCRIPTION
**Pull request by stealthanthrax**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Organises code by moving js and css files to separate folders

### CODE REVIEW

**Comment by stealthanthrax** [comment]
@zamhaq @ms10398 @gabru-md ,please have a look

**Comment by ms10398** [comment]
Whats the status @stealthanthrax

**Comment by stealthanthrax** [comment]
@ms10398 ,please have a look

**Comment by zamhaq** [comment]
@stealthanthrax update your branch and squash commits to single.

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Chrome Extension of SUSI AI",
   "dependencies": {},
   "devDependencies": {
-    "grunt": "^1.0.1",
+      "grunt": "^1.0.1",
```
Comment (zamhaq) [inline package.json]: why edit these line? @stealthanthrax

File: src/options.html [hunk 2]
```diff
@@ -1,60 +1,61 @@
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">   
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```
Comment (zamhaq) [inline src/options.html]: and what about these lines? Why are you editing them without any changes?

Comment (ms10398) [reply src/options.html]: @zamhaq I think he just removed extra space thats why its shown in edited.

**Comment by stealthanthrax** [comment]
@zamhaq @ms10398 ,this pr was closed due to an accidental force push , is there a possible way to reopen it rather than making a new pr?

**Pull request closed without merging**